stubs/pyaudio/METADATA.toml

```diff
@@ -0,0 +1,2 @@
+version = "0.1"
```

Comment:
This means the lowest version that is supported, is that correct? 0.1 is from 2006 (https://pypi.org/project/PyAudio/#history)

Reply:
Changed to the recent version number.